channeldb/invoices.go

```diff
@@ -74,6 +74,30 @@ const (
 	MaxPaymentRequestSize = 4096
 )
 
+// ContractState describes the state the invoice is in.
+type ContractState uint8
+
+const (
+	// ContractOpen means the invoice has only been created.
+	ContractOpen ContractState = 0
+
+	// ContractSettled means the htlc is settled and the invoice has been
+	// paid.
+	ContractSettled = 1
```

Comment:
I think this one also needs to be explicitly specified as the prior one isn't using `iota`?

Reply:
Fixed